projects/2023-NH/analysis.script_hadacek.R

```diff
@@ -109,24 +115,143 @@ jointdataset <- jointdataset %>%
 
 # DATA VISUALIZATION ####
 
+#self demographics 
+options(qwraps2_markup = "markdown")
+jointdataset <- as.data.frame(data_self)
+summary_statistics <-
+  list(
+    "Race" =
+      list(
+        "White" = ~qwraps2::n_perc(na.omit(race_f) %in% "1"),
+        "Black" = ~qwraps2::n_perc(na.omit(race_f) %in% "2"),
+        "Asian" = ~qwraps2::n_perc(na.omit(race_f) %in% "3"),
+        "Multiple" = ~qwraps2::n_perc(na.omit(race_f) %in% "4"),
+        "White non-Europe" = ~qwraps2::n_perc(na.omit(race_f) %in% "5"),
+        "Latinx" = ~qwraps2::n_perc(na.omit(race_f) %in% "6"),
+        "Other" = ~qwraps2::n_perc(na.omit(race_f) %in% "7"),
+      ),
+    "Gender" =
+      list(
+        "Female" = ~qwraps2::n_perc(na.omit(gender) %in% "2"),
+      ),
+    "Age" =
+      list(
+        "mean (sd)" = ~qwraps2::mean_sd(age, na_rm = TRUE),
+        "median (Q1, Q3)" = ~qwraps2::median_iqr(age, na_rm = TRUE),
+        "min" = ~min(age, na.rm = TRUE),
+        "max" = ~max(age, na.rm = TRUE),
+      )
+  )
+summary_table(data_self, summary_statistics)
+descriptive_self <- as.data.frame(summary_table(data_self, summary_statistics))
+View(descriptive_self)
```

Comment:
I assume you want to turn this dataframe into an APA style table. I'm not familiar with this, but check out this blog post if you haven't: https://www.r-bloggers.com/2017/03/generating-apa-style-tables-in-r-current-challenges/